Transcendence/Transcendence/CGameSettings.cpp

```diff
@@ -92,6 +92,15 @@ SOptionDefaults g_OptionData[CGameSettings::OPTIONS_COUNT] =
 		{	"soundVolume",				optionInteger,	"7",		0	},
 		{	"musicVolume",				optionInteger,	"7",		0	},
 		{	"musicPath",				optionString,	"",			0	},
+		
+		//	Accessibility options
+		{	"colorIFFPlayer",			optionString,	"0xFFFFFF",	0	},
```

Comment:
I would use standard hex color code like HTML/CSS: #50ff50
If you call LoadRGBColor, that comes for free.

Reply:
ah cool didnt know that was a thing, I'll look  into switching that over after taking care of some of the lower hanging fruits

Reply:
Added to LoadRGBColor, but also switched to image editor style hex codes too as the defaults in the settings file, so the functionality is there if anyone enters RGB codes using programmer style hex ('0x') just not mandatory to use